Automattic/woocommerce-payments
# Fix: #1721 Replaced column with info icon with transaction id column for CSV

Fixes #1721 

#### Changes proposed in this Pull Request
- Replaced info icon which was at the start of the table as per https://github.com/Automattic/woocommerce-payments/issues/1721#issuecomment-837114313 with Transaction Id column.
- Added a Transaction Id column with key `transaction_id` only for the CSV file.
- Updated failing snapshots and added test for `getColumns` function.

#### After fix
![image](https://user-images.githubusercontent.com/15019298/117931662-1274cc80-b31d-11eb-9d8f-54c27c710ad0.png)
![image](https://user-images.githubusercontent.com/15019298/117932624-34228380-b31e-11eb-9d6b-2db7d959842a.png)

#### Testing instructions
- Switch to `fix/1721-extra-column-csv-bug`
- Goto transactions page
- Click on `Download`
- Now it should have a new column with a label as Transaction Id.

Not sure if we require unit tests for CSV export.
-------------------
